bf2fc6cc711aee1a0c2a/kas-fleet-manager
# chore: Remove Owner field from KFM Service account API spec

## Description
The Owner field in the Service account API spec has been marked for deprecation. An email has been sent out notifying people of this deprecation. It needs to be fully removed from the code.

Jira: [Here](https://issues.redhat.com/browse/MGDSTRM-7913)

## Verification Steps
1. Run `make run/docs`
2. Can see that in the Service account endpoints the Owner field is no longer returned.

## Checklist (Definition of Done)
